src/box/sql/vdbe.c

```diff
@@ -4460,7 +4457,6 @@ default: {          /* This is really OP_Noop and OP_Explain */
 
 	/* This is the only way out of this procedure. */
 vdbe_return:
-	p->aCounter[SQL_STMTSTATUS_VM_STEP] += (int)nVmStep;
```

Comment:
Looks unrelated.

Reply:
This field is used only in the `sql_stmt_status()` function, which I decided to remove instead of refactoring, since it is not used anywhere.

Reply:
It'd be better to drop `sql_stmt_status` in a separate commit then, because this change is unrelated to dropping the `sql_stmt` alias.

Reply:
Fixed. Moved all dropped functions in the new commit.

Also, I will try to stick to the "one change = one commit" rule in the future, even with refactoring.